website/sidebars.json

```diff
@@ -28,6 +30,7 @@
       "deployment/cloudwatch-logs-delivery"
     ],
     "Configuration": [
+      "configuration/data-management-types",
```

Comment:
This change isn't working right now because `data-management-types` is already under operator docs. So we could move it here, but having it in both is not an option in this version of docusaurus. I tried to have it on both :(